sycl/include/sycl/exception.hpp

```diff
@@ -74,11 +74,21 @@ class __SYCL_EXPORT exception : public virtual std::exception {
 
   exception(std::error_code, const char *Msg);
 
+#ifdef __INTEL_PREVIEW_BREAKING_CHANGES
+  exception(std::error_code ec, const std::string &Msg)
```

Comment:
Please use single naming convention.

Reply:
Well, somebody has to stop spreading these bad names. Feel free to fix existing code in a separate PR and rebase this once that other PR is merged.

Reply:
Sure, I will change all "ec" variables into "Ec" in this file.

Reply:
Thanks. Done.